clearwater-sip-stress-coreonly.root/usr/share/clearwater/sip-stress/callee_redirection.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
+<!DOCTYPE scenario SYSTEM "sipp.dtd">
+
+<scenario name="Clearwater callee">
+  <recv request="INVITE" rrs="true" crlf="true">
+    <action>
+      <!-- Save off our identity for later use -->
+      <ereg regexp=".*<sip:(.*)@(.*)>.*" search_in="hdr" header="To:" check_it="true" assign_to="_,dn,home_domain" />
+    </action>
+  </recv>
+
+  <nop>
+    <action>
+      <!-- Save off the per-transaction headers, for use on the eventual 180 Ringing and 200 OK -->
+      <assignstr assign_to="invite_cseq" value="[last_Cseq:]" />
+      <assignstr assign_to="invite_via" value="[last_Via:]" />
+    </action>
+  </nop>
+
+  <send>
+    <![CDATA[
+      SIP/2.0 302 Moved Temporarily 
+      [last_Call-ID:]
+      [last_CSeq:]
+      [last_From:]
+      [last_To:];tag=[pid]SIPpTag01[call_number]
+      [last_Via:]
+      Server: SIP/2.0
+      Content-Length: [len]
+      [last_P-Charging-Vector:]
+      P-Asserted-Identity: sip:[$dn]@[$home_domain]
+      Contact: <sip:8888000002424@ame22.clearwater.test>;q=0.2,<sip:8888000001313@ame22.clearwater.test>;q=0.1,<sip:8888000003535@ame22.clearwater.test>;q=0.3
+      Record-Route: <sip:[local_ip]:[local_port];transport=[transport];lr>
+      [last_Record-Route:]
+      Allow: INVITE,ACK,OPTIONS,CANCEL,BYE,UPDATE,INFO,REFER,NOTIFY,MESSAGE,PRACK
+      P-Access-Network-Info: 3GPP-E-UTRAN-FDD;utran-cell-id-3gpp=0010100011a2d051
+      User-Agent: BBBBBBB IMS 0.0
+
+      v=0
```

Comment:
I still don't think we need this body, and if you have time it might be worth trying without, getting the diagnostics (so we can see what headers are missing and where the message stops, and fix whatever bug causes that), and discussing with me. If you don't have time, though, I can't find anything saying this is technically invalid, so fine to merge.